### PR TITLE
style(material/api-theme): change api page links color

### LINKED ISSUE #911: Styles for links on API page are unreadable and don't have sufficient contrast in dark mode
# Reproduction

https://material.angular.io/components/tooltip/api

Screen Shot 2021-01-29 at 13 28 40
Screen Shot 2021-01-29 at 13 30 02

Screen Shot 2021-01-29 at 13 32 22

### PR DESCRIPTION
**Pull request by kian23kpt**
Styles for links on API page was unreadable and don't have sufficient contrast in dark mode so now with change theme links color will change.

Fixes #911

### CODE REVIEW

**Comment by kian23kpt** [comment]
Now API page links color set like as theme color and have sufficient contrast.

Screenshot (12)
Screenshot (13)
Screenshot (10)
Screenshot (11)

File: src/styles/_api-theme.scss [hunk 1]
```diff
@@ -47,6 +47,11 @@
     word-break: break-word;
   }
 
+  //API docs link color will change with theme change
```
Comment (Splaktar) [inline src/styles/_api-theme.scss:50]:
```suggestion
  // API docs link color will change with theme change
```

**Review by Splaktar — COMMENTED** [review]
Thank you for this. Can you please post screenshots for both `a` and `a:visited`?

And rebase this branch on `master`?

**Comment by kian23kpt** [comment]
> 
> 
> Thank you for this. Can you please post screenshots for both `a` and `a:visited`?
> 
> And rebase this branch on `master`?

I have checked. there is no difference between them.

Screenshot (11)

**Review by Splaktar — CHANGES_REQUESTED** [review]
> I have checked. there is no difference between them.

We should probably have a slight difference between visited and unvisited links.

That said, it seems like something happened to this PR when you merged `master` into this branch. We don't accept merge commits, so we actually need this to be rebased. However, at this point, I don't see the original changes anymore. So it might be best to create a new branch and PR.

**Comment by Splaktar** [comment]
It looks like this was resolved in PR https://github.com/angular/material.angular.io/pull/990 by @crisbeto.

**Pull request closed without merging**